packages/flutter/lib/src/material/drawer_theme.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Defines default property values for descendant [Drawer] widgets.
+///
+/// Descendant widgets obtain the current [DrawerThemeData] object
+/// using `DrawerTheme.of(context)`. Instances of [DrawerThemeData] can be
+/// customized with [DrawerThemeData.copyWith].
+///
+/// Typically a [DrawerThemeData] is specified as part of the
+/// overall [Theme] with [ThemeData.drawerTheme].
+///
+/// All [DrawerThemeData] properties are `null` by default.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+@immutable
+class DrawerThemeData with Diagnosticable {
+  /// Creates a theme that can be used for [ThemeData.drawerTheme].
+  const DrawerThemeData({
+    this.backgroundColor,
+    this.scrimColor,
+    this.elevation,
+    this.shape,
+  });
+
+  /// Color to be used for the [Drawer]'s background.
```

Comment:
It would just a tiny bit more accurate to say: "Overrides the default value of [Drawer.backgroundColor]". That links this property to the full description of how the color is computed and what it means. 

Same comment for all of the properties

Reply:
Updated all comments